### PR TITLE
feat: enable session_server configuration

### LINKED ISSUE #165: Add support for session_server config
Configuration of `[session_server]` section is currently not supported. It is essential to provide [interactive web terminals](https://docs.gitlab.com/ee/ci/interactive_web_terminal/index.html).

### PR DESCRIPTION
**Pull request by Normo**
* Closes #165

### CODE REVIEW

File: tasks/update-configuration.yml [hunk 1]
```diff
@@ -20,14 +20,53 @@
...
+  notify:
+    - Restart GitLab-Runner
+
+- name: "Set session_server advertise_address value"
+  ansible.builtin.lineinfile:
+    dest: "{{ gitlab_runner_config_path }}"
+    regexp: '^(\s*)advertise_address =.*'
+    line: '  advertise_address = "{{ gitlab_runner_session_server_advertise_address }}"'
+    state: present
+    insertafter: '^\[session_server\]\n(\s*)listen_address =.*'
```
Comment (Normo) [inline tasks/update-configuration.yml:50]: @tobiashuste This does not seem to work as expected for some reason. The result is always: 
```toml
[session_server]
  listen_address = "0.0.0.0:8093"
  session_timeout = 1800
  advertise_address = "0.0.0.0:8093"
```

Comment (Normo) [reply tasks/update-configuration.yml:50]: Btw: Even when `gitlab_runner_enable_session_server` is set to `false` the runner always configures the following:
```toml
[session_server]
  session_timeout = 1800
```

I tried the suggested order `listen_address` :arrow_right: `session_timeout` :arrow_right: `advertise_address` but it doesn't make a difference.

Comment (Normo) [reply tasks/update-configuration.yml]: Now, it should be [fixed](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/9e43bee776d02f4077f2a69736211c5107c14b31..2eaa240a8b7d99115c423746f939f928ae816b14).

**Review by tobiashuste — APPROVED** [review]
Thanks, @Normo. This looks very good.

Similar to the cache implementation this does not allow the removal of the options currently. Correct?

**Comment by Normo** [comment]
> Similar to the cache implementation this does not allow the removal of the options currently. Correct?

Yes. See #175

**Pull request merged**